routingpy/routers/valhalla.py

```diff
@@ -128,7 +128,7 @@ def directions(
     ):
         """Get directions between an origin point and a destination point.
 
-        For more information, visit https://github.com/valhalla/valhalla/blob/master/docs/api/turn-by-turn/api-reference.md.
+        For more information, visit https://github.com/valhalla/valhalla/blob/master/docs/docs/api/matrix/api-reference.md.
```

Comment:
I think it should be [this link](https://github.com/valhalla/valhalla/blob/master/docs/docs/api/turn-by-turn/api-reference.md)

Reply:
Sorry, I did not even mean to fix this one, but the matrix reference one. Seems like this one was broken as well, so now both should be fixed.